_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -217,7 +219,18 @@ This `div` element is not [included in the accessibility tree][], hence its [WAI
 <div role="button" aria-sort="" style="display:none;"></div>
 ```
 
+#### Passed Example 12
```

Comment:
```suggestion
#### Inapplicable Example 3
```
I believe this is intended to be an inapplicable since it doesn't have an explicitly set ARIA attribute.